py/objint_longlong.c

```diff
@@ -40,14 +40,6 @@
 
 #if MICROPY_LONGINT_IMPL == MICROPY_LONGINT_IMPL_LONGLONG
 
-// Python3 no longer has "l" suffix for long ints. We allow to use it
```

Comment:
Right, and so the bottom line is that MICROPY_DEBUG_NLR is a one-off case, not a new pattern we follow (until it will undergo enough discussion and dubbed to be so, which discussion is not a priority).